src/log/log.c

```diff
@@ -21,6 +21,27 @@ static int __log_write_internal(
 #define	WT_LOG_OPEN_VERIFY	0x02
 
 /*
+ * __log_panic --
+ *	Report an error, then panic the system.
+ */
+static int
```

Comment:
I believe the ``WT_PANIC_MSG`` macro mirrors the functionality you added here.

I'd be happy to change our ``__wt_panic`` to require an error message instead - I think that's a reasonable requirement for an operation that intends to panic the database.

I looked and the check for ``WT_CONN_PANIC`` on API entry doesn't result in another call to ``__wt_panic``, so changing that function shouldn't result in getting an error message from every thread.